views/js/qtiCommonRenderer/helpers/Graphic.js

```diff
@@ -781,6 +781,19 @@ define([
                 }
             }
 
+            // recalculate again in case of scaled image via transform
+            var matrix = $container.closest('.transform-scale').css('transform');
```

Comment:
Please consider using the module `ui/transformer`to get the matrix values, it manage the vendor prefix also because there still browsers that use `-webkit-transform`